docs/content/en/getting_started/upgrading.md

```diff
@@ -68,6 +68,10 @@ godojo installations
 
 If you have installed DefectDojo on "iron" and wish to upgrade the installation, please see the [instructions in the repo](https://github.com/DefectDojo/godojo/blob/master/docs-and-scripts/upgrading.md).
 
+## Upgrading to DefectDojo Version 2.20.x.
+
+There are no special instruction for upgrading to 2.20.0. Check the [Release Notes](https://github.com/DefectDojo/django-DefectDojo/releases/tag/2.17.0) for the contents of the release.
```

Comment:
The link is pointing to the wrong tag.

Reply:
Copy paste is a biter :/
Thanks for catching